pkg/backend/openshiftcluster.go

```diff
@@ -235,6 +259,9 @@ func (ocb *openShiftClusterBackend) updateAsyncOperation(ctx context.Context, lo
 				asyncdoc.OpenShiftCluster = &ocCopy
 			}
 
+			clusterResult := newOperationResult(resultType, initialProvisioningState.String(), backendErr)
+			log.Printf("Long running result: %s", clusterResult.String())
```

Comment:
I am wondering, if this is the only place which uses the result, who will be reading that log?
If it is for us (SREs) I would love to see the text representation returned instead of an `int`.
If it is for analytics in the Kusto, I believe there should be no difference in creating categories based on `string` or `int`.